sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/TextAnalyticsClientImplBuilder.java

```diff
@@ -192,7 +192,7 @@ public TextAnalyticsClientImplBuilder addPolicy(HttpPipelinePolicy customPolicy)
      */
     public TextAnalyticsClientImpl buildClient() {
         if (apiVersion == null) {
-            this.apiVersion = "v3.1";
+            this.apiVersion = "v3.2-preview.1";
```

Comment:
Have we mentioned in the documentation that the SDK will default to "v3.2-preview.1" if the user does not specify any apiVersion?

Reply:
I will update it in the readme as marina mentioned